uvloop/handles/pipe.pyx

```diff
@@ -12,6 +12,9 @@ cdef __pipe_init_uv_handle(UVStream handle, Loop loop):
     err = uv.uv_pipe_init(handle._loop.uvloop,
                           <uv.uv_pipe_t*>handle._handle,
                           0)
+    # UV_HANDLE_READABLE allows calling uv_read_start() on this pipe
+    # even if it is O_WRONLY, see also #317, libuv/libuv#2058
+    handle._handle.flags |= 0x00004000  # UV_HANDLE_READABLE
```

Comment:
Can you import the `UV_HANDLE_READABLE` constant via `uv.pxd`?

Reply:
Oh sure! I can move the magic number into `uv.pxd` - but it'll be still a magic number in uvloop, because it's an internal flag in libuv, we cannot import it (or maybe I missed anything?).

Reply:
Ah, I see. I'd then still add it to `uv.pxd` and maybe name it something like `UV_INTERNAL_HANDLE_READABLE` and add a nice comment to it.